src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/items/illuminant_steel_ingot.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "Illuminant Steel Ingot",
+    "icon": "minebound:illuminant_steel_ingot",
+    "category": "minebound:alloys",
+    "pages": [
+        {
+            "type": "patchouli:text",
+            "text": "$(o)Steel is iron on coke, and thanks to the innately magical properties of the nether's very own light source, alloying coal and glowstone with energized iron results in an even stronger metal with better energy conduction!"
+        }, 
+        {
+            "type": "patchouli:text",
+            "text": "Craft in an $(l:minebound:blocks/alloy_furnace)alloy furnace$() by smelting an $(l:minebound:items/energized_iron) with coal and glowstone dust."
```

Comment:
The `minebound:items/energized_iron` link is bad.

Reply:
fixed